lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -21,7 +20,7 @@
 %if settings.FEATURES.get('ENABLE_S3_GRADE_DOWNLOADS'):
   <div class="reports-download-container action-type-container">
     <hr>
-    <h2> ${_("Reports")}</h2>
+    <h3 class="hd hd-2">${_("Reports")}</h3>
```

Comment:
I had to update the headings to follow proper hierarchy. There's an H2 on line 4, which represents the content of the section, so the next logical heading level is 3.